boards/arm/arduino_nano_33_ble/arduino_nano_33_ble.dts

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2020 Jefferson Lee
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+    model = "Arduino Nano 33 BLE (Sense)";
+    compatible = "arduino,arduino_nano_33_ble";
+
+    chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &code_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio0 24 0>;
+			label = "Red LED";
+		};
+	};
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		led0 = &led0;
+		spi = &spi2;
+	};
+};
+
+&flash0 {
+	/*
+	 * For more information, see:
+	 * https://docs.zephyrproject.org/latest/guides/dts/legacy-macros.html#legacy-flash-partitions
+	 * these bindings are irrelevant. See the LD script
+	 */
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		code_partition: partition@10000 {
+			label = "code";
+			reg = <0x10000 0xf0000>;
+			read-only;
+		};
+	};
+};
+
+/*
+ * guide specifying pin numbers: https://docs.zephyrproject.org/latest/boards/arm/nrf52840dk_nrf52840/doc/index.html
+ * Given Pin PB.A:
+ * pin_no = B*32 + A
+ * ex. P1.0 => 32
+*/
+&adc {
+	status = "okay";
+};
+&uart0 {
+	compatible = "nordic,nrf-uart";
+	current-speed = <115200>;
+	status = "okay";
+	tx-pin = <35>; //P1.03
+	rx-pin = <42>; //P1.10
+};
+&i2c0 {
```

Comment:
Looking at the schematics you may want to swap these as the sensors are connected to pins 14/15.